### PR TITLE
Add popup window functionality to work-section buttons

### PR DESCRIPTION
**Pull request by mullenlucas**
# Developed functionalities:
- Created windows to popup when clicking the "See more" button on the work-section's card.
- Added style for said popup in .css file for mobile and desktop visualizations.

## popups.js
- Created popups.js to store data on the information of each of the work-section cards.

## script.js
- Deploy the html structure dynamically when clicking on a work-section button.
    - Clicking on one of these buttons triggers the creation of html element and structure according to the screen size.
- Added close functionality to the popup window, both when clicking on the exit button or clicking outside the popup.
   - Closing the window removes the newly created html elements when clicking a work-section button.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -94,7 +95,7 @@ <h3>Multi-Post Stories Gain+Glory</h3>
 						</ul>
 					</div>
 					<div class="card-see-prjct">
-						<button class="green-button" type="button">See Project</button>
+						<button class="green-button cProject" id="cP1" type="button">See Project</button>
 					</div>
 					</div>
 			</div>
```
Comment (cwaku) [inline index.html:101]: - [ ] The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.**

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @mullenlucas 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
There are a few issues 🐛  that you still need to work 🛠️  on to go to the next project ⏭️  but you are almost there! 🏃🏾 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 

## Required Changes ♻️
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips 🌟  that can improve your code quality 💯 and help you get a better job 💼.

I will guide 📔 you through fixing 🛠️ these bugs, follow my comments down below and make sure you don't miss any of them, in order to get your project approved ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mullenlucas) [reply index.html]: Thanks for the suggestion, I made the works-section content generate dynamically on javascript.
Added a "reducedBadges" list on the `popupData` that contains the main badges of each card so as to keep the card style consistent

**Review by Ismailco — APPROVED** [review]
### Hi @mullenlucas, @cwaku, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**